recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "featurewiz" %}
-{% set version = "0.1.91" %}
+{% set version = "0.1.99" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/featurewiz-{{ version }}.tar.gz
-  sha256: 54369a1bda44d2b870ebd971b67677a90e4fa7118893bdbda04ec184c8039d46
+  sha256: 25f6f7cbcdb605d706e4e1f61a85e02111c1810b3f4b688e07e921d28b3ea3e3
```

Comment:
```suggestion
  sha256: b74bd71b45f950fb2ffdab8ea3cfbbc585fd0f6fef80c57f928df8e4ca0eda45
```

Reply:
Which I obtained by running 
```
 curl -sL https://pypi.io/packages/source/f/featurewiz/featurewiz-0.2.4.tar.gz | openssl sha256
```

Reply:
Thanks @tdegeus

Reply:
So this is crucial to change with the version, see the error in the CI